flake8_quotes.py

```diff
@@ -1,8 +1,11 @@
+import os
 import tokenize
 
 import pep8
 
-__version__ = '0.2.2'
+
+with open(os.path.join(os.path.dirname(__file__), 'VERSION')) as handle:
```

Comment:
`r` should be fine; there is no special encoding being used on `VERSION` and we own it. I don't think `rt` is Python 2 compatible.